<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(31,31,31)
}</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QFrame" name="sidebar_container">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>720</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="sidebar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>720</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>321</width>
       <height>720</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color: rgb(40,40,40)
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="header1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(220,220,220)
}</string>
      </property>
      <property name="text">
       <string>SHORTEST ROUTE</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="header2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(176,153,141)
}</string>
      </property>
      <property name="text">
       <string>FINDER</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="import_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(143,115,101);
color: rgb(220,220,220);
border: none
}

QPushButton:hover {
background-color: rgb(176,153,141)
}</string>
      </property>
      <property name="text">
       <string>Import File</string>
      </property>
     </widget>
     <widget class="QLabel" name="file_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color : rgb(220,220,220);
background-color: rgb(31,31,31);
padding-left: 5;
padding-right: 5;
border: 1px solid rgb(143,115,101)
}</string>
      </property>
      <property name="text">
       <string>No file</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(154,131,101);
padding-left: 5px;
border: 1px solid rgb(154,131,101)
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="start_input">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(31,31,31);
color: rgb(220,220,220);
border: 1px solid rgb(143,115,101);
padding-left: 5;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="end_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(143,115,101);
border: 1px solid rgb(143,115,101);
border-right: 0px;
padding-left: 5px
}</string>
      </property>
      <property name="text">
       <string>End</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="end_input">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(31,31,31);
color: rgb(220,220,220);
border: 1px solid rgb(143,115,101);
padding-left: 5
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(143,115,101);
color: rgb(220,220,220);
border: none;
}

QPushButton:hover {
background-color: rgb(176,153,141)
}</string>
      </property>
      <property name="text">
       <string>Search!</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
color: rgb(220,220,220)
}

QRadioButton::indicator {
background-color: rgb(31,31,31);
border: 1px solid rgb(143,115,101);
width: 18;
height: 18;
}

QRadioButton::indicator::unchecked {
   background-color: rgb(31,31,31);
}

QRadioButton::indicator::checked {
   background-color: rgb(143,115,101);
}</string>
      </property>
      <property name="text">
       <string>UCS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
color: rgb(220,220,220)
}

QRadioButton::indicator {
background-color: rgb(31,31,31);
border: 1px solid rgb(143,115,101);
width: 18;
height: 18;
}

QRadioButton::indicator::unchecked {
   background-color: rgb(31,31,31);
}

QRadioButton::indicator::checked {
   background-color: rgb(143,115,101);
}</string>
      </property>
      <property name="text">
       <string>A*</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>690</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(190,190,190)
}</string>
      </property>
      <property name="text">
       <string>Ver 1.0.0</string>
      </property>
     </widget>
     <zorder>file_label</zorder>
     <zorder>header1</zorder>
     <zorder>header2</zorder>
     <zorder>import_btn</zorder>
     <zorder>start_label</zorder>
     <zorder>start_input</zorder>
     <zorder>end_label</zorder>
     <zorder>end_input</zorder>
     <zorder>search_btn</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QFrame" name="sidebar_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color: rgb(41,41,41);
border-radius: 25px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>13</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: url(&quot;src/asset/setting.png&quot;);
background-repeat: none;
background-position: center;
border-radius: 25px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="solution">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>440</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
background-color: rgb(41,41,41)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="solution_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: rgb(220,220,220)
}</string>
     </property>
     <property name="text">
      <string>Shortest Route</string>
     </property>
    </widget>
    <widget class="QLabel" name="distance_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(154,131,101);
padding-left: 5px;
border: 1px solid rgb(154,131,101)
}</string>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
    <widget class="QLabel" name="distance_value">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(220,220,220);
padding-left: 5px;
border: 1px solid rgb(154,131,101)
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="route_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(154,131,101);
padding-left: 5px;
border: 1px solid rgb(154,131,101)
}</string>
     </property>
     <property name="text">
      <string>Route</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="route_value">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
background-color: rgb(31,31,31);
color: rgb(220,220,220);
padding: 5px;
border: 1px solid rgb(154,131,101)
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
